diff -rupN denemo-1.0.2/src/main.c denemo-1.0.2.new/src/main.c
--- denemo-1.0.2/src/main.c	2013-05-06 20:43:44.812271925 -0500
+++ denemo-1.0.2.new/src/main.c	2013-05-13 09:59:24.765015382 -0500
@@ -256,6 +256,14 @@ debug_handler (const gchar *log_domain,
   //g_debug ("%s",message);
 }
 
+#ifdef G_OS_WIN32
+    #define PATH_SEPERATOR ";"
+    #define DYLIB_EXT ".dll"
+  #else
+    #define PATH_SEPERATOR ":"
+    #define DYLIB_EXT ".so"
+#endif
+
 static
 void append_to_path(gchar *path, gchar *extra) {
   gchar * the_path = (gchar*)g_getenv (path);
@@ -348,6 +356,44 @@ main (int argc, char *argv[])
   append_to_path ("GUILE_LOAD_PATH", g_build_filename(prefix, "share", "denemo", NULL));
 
 #else
+  gchar *prefix = g_build_filename (g_path_get_dirname(get_bin_dir()), "..", NULL);
+#endif
+  g_print ("PREFIX==%s\n", prefix);
+
+  g_setenv ("GTK_MODULE_VERSION", "2.10.0", TRUE);
+  g_print ("Setting GTK_MODULE_VERSION=2.10.0\n");
+  g_setenv ("GTK_SO_EXTENSION", DYLIB_EXT, TRUE);
+  g_print ("Setting GTK_SO_EXTENSION==%s\n", DYLIB_EXT);
+  g_setenv ("GTK_PREFIX", prefix, TRUE);
+  g_print ("Setting GTK_PREFIX=%s\n", prefix);
+
+  gchar *gdk_module_dir = g_build_filename (prefix, "lib", "gtk-2.0", "2.10.0", "loaders", NULL);
+  g_setenv ("GDK_PIXBUF_MODULEDIR", gdk_module_dir, TRUE);
+  g_print ("Setting GDK_PIXBUF_MODULEDIR=%s\n", gdk_module_dir);
+  gchar *gdk_pixbuf_module_file = g_build_filename (prefix, "etc", "gtk-2.0" , "gdk-pixbuf.loaders", NULL);
+  g_setenv ("GDK_PIXBUF_MODULE_FILE", gdk_pixbuf_module_file, TRUE);
+  g_print ("Setting GDK_PIXBUF_MODULE_FILE=%s\n", gdk_pixbuf_module_file);
+
+  gchar *fc_path = g_build_filename (prefix, "etc","fonts", NULL);
+  g_setenv ("FONTCONFIG_PATH", fc_path, TRUE);
+  g_print ("Setting FONTCONFIG_PATH=%s\n", fc_path);
+  gchar *fc_file = g_build_filename (fc_path, "fonts.conf", NULL);
+  g_setenv ("FONTCONFIG_FILE", fc_file, TRUE);
+  g_print ("Setting FONTCONFIG_FILE=%s\n", fc_file);
+
+  gchar *fontpath = g_build_filename (prefix, "share", "fonts", "truetype","denemo", "feta.ttf", NULL);
+  g_setenv ("LILYPOND_VERBOSE", "1", FALSE);
+  g_print("LILYPOND_VERBOSE=1\n");
+  add_font_file(fontpath);
+  fontpath = g_build_filename (prefix, "share", "fonts", "truetype","denemo", "Denemo.ttf", NULL);
+  add_font_file(fontpath);
+  fontpath = g_build_filename (prefix, "share", "fonts", "truetype","denemo", "emmentaler.ttf", NULL);
+  add_font_file(fontpath);
+  //probably not necessary to append to guile path
+  //append_to_path ("GUILE_LOAD_PATH", g_build_filename(prefix, "share", "denemo", NULL));
+
+#if 0
+
   gchar *prefix = g_build_filename (get_prefix_dir(), NULL);
   add_font_directory (g_build_filename (get_data_dir(), "/fonts", NULL));
 
@@ -362,8 +408,11 @@ main (int argc, char *argv[])
   append_to_path("GUILE_LOAD_PATH", g_build_filename(prefix, "share", "denemo", "actions", NULL));  
   append_to_path("GUILE_LOAD_PATH", g_build_filename(prefix, "share", "denemo", "actions", "denemo-modules", NULL));  
 
+
+  
 #endif /* end of else not windows */
 
+
   g_setenv ("LYEDITOR", "denemoclient %(line)s %(column)s", FALSE);
   GError *error = NULL;
   /* glib/gtk initialization */
