diff --git a/src/print.c b/src/print.c
index 43efd90..96cde60 100644
--- a/src/print.c
+++ b/src/print.c
@@ -462,11 +462,11 @@ run_lilypond (gchar ** arguments)
                                                                &get_print_status()->printpid,
                                                                NULL,
                                                                NULL,    /* stdout */
-#ifdef G_OS_WIN32
+//#ifdef G_OS_WIN32
                                                                NULL,
-#else
-                                                               &LilyPond_stderr,        /* stderr */
-#endif
+//#else
+//                                                               &LilyPond_stderr,        /* stderr */
+//#endif
                                                                &lily_err);
   if (lily_err)
     {
