diff -rupN denemo-1.0.0/src/prefops.c denemo-1.0.0.new/src/prefops.c
--- denemo-1.0.0/src/prefops.c	2013-03-22 22:42:16.000000000 -0500
+++ denemo-1.0.0.new/src/prefops.c	2013-03-24 15:08:34.859876111 -0500
@@ -105,7 +105,7 @@ initprefs ()
 #else /* !G_OS_WIN32 */
   ret->browser = g_string_new ("firefox");
   ret->audioplayer = g_string_new ("play");
-  ret->lilypath = g_string_new ("lilypond");
+  ret->lilypath = g_string_new (g_build_filename(get_bin_dir(), "lilypond", NULL));
   ret->pdfviewer = g_string_new ("evince");
   ret->imageviewer = g_string_new ("eog");
 #endif /* !G_OS_WIN32 */
