diff -rupN evince-2.32.0/cut-n-paste/Makefile.in evince-2.32.0.new/cut-n-paste/Makefile.in
--- evince-2.32.0/cut-n-paste/Makefile.in	2010-09-27 15:02:24.000000000 -0400
+++ evince-2.32.0.new/cut-n-paste/Makefile.in	2015-01-26 00:30:13.000000000 -0500
@@ -324,7 +324,7 @@ target_alias = @target_alias@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = zoom-control toolbar-editor totem-screensaver smclient gimpcellrenderertoggle synctex
+SUBDIRS = zoom-control toolbar-editor totem-screensaver gimpcellrenderertoggle synctex
 all: all-recursive
 
 .SUFFIXES:
diff -rupN evince-2.32.0/Makefile.am evince-2.32.0.new/Makefile.am
--- evince-2.32.0/Makefile.am	2010-07-27 11:47:54.000000000 -0400
+++ evince-2.32.0.new/Makefile.am	2015-01-26 22:16:16.000000000 -0500
@@ -13,7 +13,6 @@ SUBDIRS = \
 	libview \
 	libmisc \
 	properties \
-	shell \
 	po \
 	help
 
diff -rupN evince-2.32.0/Makefile.in evince-2.32.0.new/Makefile.in
--- evince-2.32.0/Makefile.in	2010-09-27 15:02:31.000000000 -0400
+++ evince-2.32.0.new/Makefile.in	2015-01-26 22:23:37.000000000 -0500
@@ -104,7 +104,7 @@ AM_RECURSIVE_TARGETS = $(RECURSIVE_TARGE
 ETAGS = etags
 CTAGS = ctags
 DIST_SUBDIRS = cut-n-paste data libdocument backend libview libmisc \
-	properties shell po help test thumbnailer previewer
+	properties help test thumbnailer previewer
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
 distdir = $(PACKAGE)-$(VERSION)
 top_distdir = $(distdir)
@@ -380,7 +380,7 @@ ACLOCAL_AMFLAGS = -I m4
 # Set the minimum required Windows version to Windows 2000 (0x0500).
 @PLATFORM_WIN32_TRUE@AM_CPPFLAGS = -D_WIN32_WINNT=0x0500
 SUBDIRS = cut-n-paste data libdocument backend libview libmisc \
-	properties shell po help $(am__append_1) $(am__append_2) \
+	properties help $(am__append_1) $(am__append_2) \
 	$(am__append_3)
 NULL = 
 pkgconfigdir = $(libdir)/pkgconfig
